docs/docsite/rst/playbook_guide/playbooks_tags.rst

```diff
@@ -192,6 +215,12 @@ or:
        # using YAML shorthand, this is equivalent to:
        # - { role: foo, tags: ["bar", "baz"] }
 
+
+.. note::
+    When adding a tag at the role level, not only are all tasks tagged, but the role's handlers are also tagged as well as everything in dependencies.
```

Comment:
While handlers are, I think, tagged the tags on handlers are ignored. I have a PR to add support for tags on handlers but that is not how it currently (or ever) works.

(Might be a language barrier on my end) but this reads that handlers are tagged and you can exclude them via `--tags`/`--skip-tags`? Which is not correct.

Reply:
I could have sworn this did work at one point, but I trust you more than my memory on this subject. Do handlers ignore tags even if directly set on the handler?

Reply:
Yeah, we don't call `filter_tagged_tasks` on them like we do on regular tasks when inserting them into `PlayIterator`.

The original `StrategyBase.run_handlers()` didn't do that too.

Reply:
updated to reflect this.